### PR TITLE
S05block some vids

### PR DESCRIPTION
**Pull request by Maachino**
Block some Vids

### CODE REVIEW

File: frontend/src/App.css [hunk 1]
```diff
@@ -120,6 +120,10 @@ button {
   text-align: center;
 }
 
+.LogoF {
```
Comment (Justine-Merlin) [inline frontend/src/App.css]: no PascalCase in css

File: frontend/src/pages/Search.jsx [hunk 2]
```diff
@@ -17,18 +21,29 @@ function Search() {
...
-                  className="imgCategory"
-                  src={video.thumbnail_url}
-                  alt={video.title}
-                />
-              </Link>
-            </div>
-          ))}
+          {resultsVideos.map((video) =>
+            video.is_public || (!video.is_public && token) ? (
+              <div key={video.id} className="thumbnail">
```
Comment (Justine-Merlin) [inline frontend/src/pages/Search.jsx]: useless `<div>`

File: frontend/src/pages/Search.jsx [hunk 3]
```diff
@@ -17,18 +21,29 @@ function Search() {
...
+                  <h2 className="legend">{video.title}</h2>
+                  <img
+                    className="imgCategory"
+                    src={video.thumbnail_url}
+                    alt={video.title}
+                  />
+                </Link>
+              </div>
+            ) : (
+              <div key={video.id} className="thumbnail">
```
Comment (Justine-Merlin) [inline frontend/src/pages/Search.jsx]: Same here

**Review by Justine-Merlin — CHANGES_REQUESTED** [review]

File: frontend/src/App.css (hunk 1 above)
Comment (Maachino) [reply frontend/src/App.css]: fixed

File: frontend/src/pages/Search.jsx (hunk 2 above)
Comment (Maachino) [reply frontend/src/pages/Search.jsx]: fixed

File: frontend/src/pages/Search.jsx (hunk 3 above)
Comment (Maachino) [reply frontend/src/pages/Search.jsx]: fixed

**Review by Justine-Merlin — APPROVED** [review]

**Pull request merged**